docker-compose.yml

```diff
@@ -851,7 +851,7 @@ services:
     hostname: exc1
     <<: *x-logging
     stop_signal: SIGINT
-    image: openruntimes/executor:0.6.1
+    image: openruntimes/executor:0.6.2
```

Comment:
I think this also needs updating in the docker-compose phtml

Reply:
Nice catch